.golangci.yml

```diff
@@ -54,6 +54,7 @@ linters:
     - goconst
     - gocyclo
     - goerr113
+    - goimports # using gci
```

Comment:
Huh? Do I need to install this locally?

Reply:
No - it is built into golangci-lint. If you get lint errors, you can fix with `golangci-lint run --fix`. We're using this in protocompile and connect-go already.

Reply:
Never noticed tbh, I don't have this installed locally.

```
zsh: command not found: golangci-lint
```

As a vs code user I rely on the native IDE tools installed for gopls to work, I guess I need to install this custom thing to have the same imports as rest of folks contributing to this repository?

```
  gotests
  gomodifytags
  impl
  goplay
  dlv
  staticcheck
  gopls
```

Reply:
I think this is identical to running `goimports -local github.com/bufbuild/plugins`.

Reply:
LOL sorry, I must have missed the boat. I see there was an entire conversation on this. Let's not spend any more time on this. 

I'll figure this out. I literally have never thought about imports, to me it's like generated code that gets re-shuffled each time I client save and I have zero opinion. I just always thought goimports was the lowest common denominator.